include/mxnet/c_api.h

```diff
@@ -1925,6 +1927,31 @@ MXNET_DLL int MXKVStoreSetUpdaterEx(KVStoreHandle handle,
                                     MXKVStoreUpdater updater,
                                     MXKVStoreStrUpdater str_updater,
                                     void *updater_handle);
+/*!
+ * \brief user-defined merger for the kvstore
+ * It's this updater's responsibility to delete \a recv and \a merged
+ * \param the key
+ * \param recv the pushed value on this key
+ * \param merged the current merged value stored on local on this key
+ * \param req_meta the metadata associated with the push request
+ * \param handle The additional handle to the merger
+ * \return 0 when push accepted, -1 otherwise
+ */
+typedef int (MXKVStoreMerger)(int key,
+                              NDArrayHandle recv,
+                              NDArrayHandle merged,
+                              mxnet::KVStore::MergeInfo info,
```

Comment:
this is kind of big, would it be better to pass by pointer?

Reply:
not requiring this to be changed, just asking

Reply:
it's just five ints right now, but it might become larger in the future. I'll make it a pointer. Thanks!